packages/encoding/src/index.ts

```diff
@@ -1,12 +1,63 @@
 /**
  * Code modified from https://github.com/github/webauthn-json/blob/e932b3585fa70b0bd5b5a4012ba7dbad7b0a0d0f/src/webauthn-json/base64url.ts#L23
  */
+export const DEFAULT_JWK_MEMBER_BYTE_LENGTH = 32;
+
 export function stringToBase64urlString(input: string): string {
   // string to base64 -- we do not rely on the browser's btoa since it's not present in React Native environments
   const base64String = btoa(input);
   return base64StringToBase64UrlEncodedString(base64String);
 }
 
+export function base64urlToBuffer(baseurl64String: string): ArrayBuffer {
+  // Base64url to Base64
+  const padding = "==".slice(0, (4 - (baseurl64String.length % 4)) % 4);
+  const base64String =
+    baseurl64String.replace(/-/g, "+").replace(/_/g, "/") + padding;
+
+  // Base64 to binary string
+  const str = atob(base64String);
```

Comment:
Generally we should avoid relying on things that are "magically" available. This may work in Node but might not work in React Native, for example (I'd like our core "encoding" functions to be pure JS so we don't have to deal with polyfills or worry about different implementations across JS runtimes)

Reply:
That makes sense! This also was cut/pasted from `packages/http/src/webauthn-json/base64url.ts` / `packages/webauthn-stamper/src/webauthn-json/base64url.ts` . But if this is "over-consolidation", happy to take this out. That and the fact that this isn't pure JS